examples/trace/index.js

```diff
@@ -0,0 +1,47 @@
+var debug = require('debug')('trace-example');
+var express = require('../../');
+
+
+function random() {
```

Comment:
Definitely a matter of taste :). But I don't want to break the Express style, so I'll change it.